universal-application-tool-0.0.1/app/services/applicant/ReadOnlyApplicantProgramServiceImpl.java

```diff
@@ -60,11 +61,10 @@ protected ReadOnlyApplicantProgramServiceImpl(
     return getBlockAfter(block.getId());
   }
 
-  // TODO(https://github.com/seattle-uat/universal-application-tool/issues/224): Implement checking
-  // blocks for completion
   @Override
   public Optional<Block> getFirstIncompleteBlock() {
-    ImmutableList<Block> currentBlockList = getCurrentBlockList();
-    return currentBlockList.isEmpty() ? Optional.empty() : Optional.of(currentBlockList.get(0));
+    return getCurrentBlockList().isEmpty()
```

Comment:
Thanks! I'm not sure this will work here though - `getCurrentBlockList` returns an `ImmutableList`, not an `Optional`. But I didn't know about map, I'm excited to use it in the future!